app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -29,6 +29,7 @@ def create_with_dynflow
             @taxonomy = Organization.new(resource_params)
             sync_task(::Actions::Katello::Organization::Create, @taxonomy)
             @taxonomy.reload
+            session[:organization_id] = @taxonomy.id
```

Comment:
But if you redirect to `select` from this action, it redirects you [back](https://github.com/theforeman/foreman/blob/develop/app/controllers/concerns/foreman/controller/taxonomies_controller.rb#L103) to /organizations/new no matter what. The aim here is switch to new org as a side effect of creating it.

> By the way, that action contains actually all the code necessary to switch to new org.

True, it does 2 things:

* sets `organization_id` into session - I do that here as well
* sweeps `TopbarCache` so that tobpar gets updated - I do not think I need it because org in tobpar is set to what I expect when I leave the wizard